content/blog/_posts/2021-02-fluvio-cloud.md

```diff
@@ -10,158 +10,140 @@ url: /blog/2021/02/announcing-fluvio-cloud-platform
-stand up and tear down whole clusters. The CLI natively supports multiple profiles,
-so you can interact with multiple clusters just by switching the active profile. This can
-be useful when working with multiple environments such as dev/test/prod or local/cloud.
-
-### Language Native APIs
-
-Our programmatic APIs are built to leverage the idioms of each language. Our Rust client
-offers a fully async interface and makes use of the ecosystem's `Future` and `Stream` traits
-to integrate smoothly with other async streaming code. In Node, we make use of `AsyncIterator`
-in order to enable `for await` syntax for iterating over records. We will be adding more
-language clients in the future, and we're always open to feedback. Feel free to join our
-[Discord chat] and let us know what language support you'd like to see.
-
-### Declarative Management
-
-Our client APIs make Fluvio a great choice for developers, but how OPs friendly is it?
-Fluvio is built to run natively on Kubernetes, employing operator patterns in order to
-self-govern cluster components and keep the system healthy. Fluvio is configured declaratively,
-meaning that human operators simply declare the target state of the system, and the cluster
-handles the rest. This also includes continuous state reconciliation - if a node goes down
-or if the state of the system diverges from the target state, Fluvio will automatically take
-action to return to the target state, meaning fewer midnight pager-duty alerts.
-
-# Conclusion
-
-Thanks for reading to the end! We hope that you're as excited as we are for
-the future of real-time applications. If you liked the blog, feel free to
-share it around or to come let us know your thoughts! We have a community
-[Discord chat] if you'd like to pop in. If you want to get started building
-with Fluvio, the easiest way to begin is with a [free Fluvio Cloud account]
-and to follow our [getting started guide]!.
-
-[Discord chat]: https://discordapp.com/invite/bBG2dTz
-[free Fluvio Cloud account]: https://cloud.fluvio.io/signup
-[getting started guide]: https://www.fluvio.io/docs/getting-started/
+One of the cool things about Fluvio is that we leverage [Rust's FFI] allowing
+us to reuse core compenents in our non-rust client libraries. This is how we
+were able to build a [node-js client] without too much work.
+
+There are tons of reasons why Fluvio is awesome but this is a post about using
+Fluvio Cloud. :)
+
+[Rust's FFI]: https://doc.rust-lang.org/nomicon/ffi.html#calling-foreign-functions
+[Fluvio CLI]: /docs/getting-started/
+[node-js client]: https://github.com/infinyon/fluvio-client-node
+
+# Using The Fluvio Cloud Platform
+
+We've written plenty of [Fluvio tutorials](/tutorials) but we'd like to show
+how Fluvio cloud acts the same as if you were using it locally or if you host
+it yourself.
+
+## Setup
+Setting up the cloud is very straight forward as mentioned in the introduction
```

Comment:
[Replace sentence with...]
Setting up Fluvio cloud is straight forward, just add your username and password and your account is provisioned in matter of seconds.